### PR TITLE
NR-89144 Apache Hadoop quickstart

### PR DESCRIPTION
**Pull request by pkudikyala**
**Ticket URL:** https://issues.newrelic.com/browse/NR-89144

**Description:** This PR has the new quickstart integration named "Apache Hadoop"

**Apache Hadoop Dashboard:** https://onenr.io/0KQXGV1vLja

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

_The owners of this repo are not experts in the subject matter of the quickstarts. We review for the quickstart to be functional and for security risks. If you are seeking feedback on the content of the quickstart, please seek out a subject matter expert. If you are not an internal NR contributor, we can do our best to connect you with a content reviewer._

## Pre merge checklist

- [ ] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [ ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [ ] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [ ] Does the PR contain a screenshot for each of your dashboards?
- [ ] Do your screenshots show data?
- [ ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by mdumpati — APPROVED** [review]
LGTM

File: quickstarts/apache-hadoop/config.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+
+slug: apache-hadoop
+description: |
+  ## Comprehensive monitoring quickstart for Apache Hadoop
+  By integrating Apache Hadoop with New Relic, you can get comprehensive information about the workflow by visualizing and gathering metrics and logs. Apache Hadoop metrics are statistical data that Hadoop daemons offer and are used for performance adjustment, debugging, and monitoring. Numerous default metrics are accessible and are highly helpful for debugging.
```
Comment (pkosanam) [inline quickstarts/apache-hadoop/config.yml:5]:
```suggestion
  Apache Hadoop metrics are statistical data that Hadoop daemons offer and are used for performance adjustment, debugging, and monitoring. By integrating Apache Hadoop with New Relic, you can get comprehensive information about the workflow by visualizing and gathering metrics and logs.
```

File: quickstarts/apache-hadoop/config.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+
+slug: apache-hadoop
+description: |
+  ## Comprehensive monitoring quickstart for Apache Hadoop
+  By integrating Apache Hadoop with New Relic, you can get comprehensive information about the workflow by visualizing and gathering metrics and logs. Apache Hadoop metrics are statistical data that Hadoop daemons offer and are used for performance adjustment, debugging, and monitoring. Numerous default metrics are accessible and are highly helpful for debugging. 
+  
+  ## Why monitor Apache Hadoop?
+  Apache Hadoop is an open-source software package which enables distributed storage and processing of massive data by leveraging a network of computers. It utilizes the MapReduce programming model-based distributed storage and processing of massive data.
+
+  By leveraging New Relic, you can gain a more in-depth understanding of Apache Hadoop performance and health. Monitoring Apache Hadoop with New Relic will help you better understand the HDFS (Hadoop Distributed File System), blocks, the system load, the data nodes, the NodeManager, and jobs.
```
Comment (pkosanam) [inline quickstarts/apache-hadoop/config.yml:10]:
```suggestion
  By leveraging New Relic, you can gain a more in-depth understanding of Apache Hadoop performance and health. Monitoring Apache Hadoop with New Relic will help you better understand the HDFS (Hadoop Distributed File System), blocks, system load, data nodes, NodeManager, and jobs.
```

File: quickstarts/apache-hadoop/config.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  ## What’s included in this quickstart?
+  New Relic Apache Hadoop monitoring quickstart ability to cover quality on out-of-the-box reporting.
```
Comment (pkosanam) [inline quickstarts/apache-hadoop/config.yml:13]:
```suggestion
  New Relic Apache Hadoop monitoring quickstart provides quality out-of-the-box reporting:
```

File: quickstarts/apache-hadoop/config.yml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
```
Comment (pkosanam) [inline quickstarts/apache-hadoop/config.yml:14]: can remove this line

File: quickstarts/apache-hadoop/config.yml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  - Jvm metrics
```
Comment (pkosanam) [inline quickstarts/apache-hadoop/config.yml:15]:
```suggestion
  - JVM metrics
```

File: quickstarts/apache-hadoop/config.yml [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  - NameNode metrics
+  - DataNode metrics
+  - Cluster metrics
+  - Queue metrics
+  - Node manager metrics
+  - Metrics system
```
Comment (pkosanam) [inline quickstarts/apache-hadoop/config.yml:21]:
```suggestion
  - Infrastructure metrics
```

**Review by pkosanam — CHANGES_REQUESTED** [review]

File: alert-policies/apache-hadoop/NameNodeDeadDataNodes.yml [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# Name of the alert
+name: NameNode Dead DataNodes
```
[the remark this answers is not in the document]
Comment (pkudikyala) [reply alert-policies/apache-hadoop/NameNodeDeadDataNodes.yml]: Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...

File: alert-policies/apache-hadoop/UsedDiskPercentage.yml [hunk 8]
```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: Used Disk Percent
+
+# Description and details
+description: |+
+  This alert is triggered when Used disk space above 90% for at least 5 minutes.
```
[the remark this answers is not in the document]
Comment (pkudikyala) [reply alert-policies/apache-hadoop/UsedDiskPercentage.yml]: Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...

File: alert-policies/apache-hadoop/NameNodeVolumeFailures.yml [hunk 9]
```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: NameNode Volume Failures
+
+# Description and details
+description: |+
+  This alert is triggered if the Volume Failures exceeds 2 for 5 minutes.
```
[the remark this answers is not in the document]
Comment (pkudikyala) [reply alert-policies/apache-hadoop/NameNodeVolumeFailures.yml]: Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...

File: alert-policies/apache-hadoop/NameNodeMissingBlocks.yml [hunk 10]
```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: NameNode Missing Blocks
+
+# Description and details
+description: |+
+  This alert is triggered if the Missing Blocks exceeds 3 for 5 minutes.
```
[the remark this answers is not in the document]
Comment (pkudikyala) [reply alert-policies/apache-hadoop/NameNodeMissingBlocks.yml]: Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...

File: quickstarts/apache-hadoop/config.yml (hunk 2 above)
Comment (pkudikyala) [reply quickstarts/apache-hadoop/config.yml]: Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...

File: quickstarts/apache-hadoop/config.yml (hunk 4 above)
Comment (pkudikyala) [reply quickstarts/apache-hadoop/config.yml]: Hi @pkosanam,
As per your suggestion, I have removed the empty line. kindly review it.

Thank you...

File: quickstarts/apache-hadoop/config.yml (hunk 3 above)
Comment (pkudikyala) [reply quickstarts/apache-hadoop/config.yml]: Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...

File: quickstarts/apache-hadoop/config.yml (hunk 5 above)
Comment (pkudikyala) [reply quickstarts/apache-hadoop/config.yml]: Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...

File: quickstarts/apache-hadoop/config.yml (hunk 6 above)
Comment (pkudikyala) [reply quickstarts/apache-hadoop/config.yml]: Hi @pkosanam,
As per your suggestion, I have made changes. kindly review it.

Thank you...

**Review by rahulbasu-nr — APPROVED** [review]
Looks good to me.

**Review by seemantk — APPROVED** [review]

**Review by mdumpati — APPROVED** [review]
LGTM

**Review by pkosanam — APPROVED** [review]
LGTM

**Pull request merged**